package.json

```diff
@@ -1,137 +1,43 @@
-    "@types/ember__runloop": "^3.16.3",
-    "@types/ember__service": "^3.16.1",
-    "@types/ember__string": "^3.16.3",
-    "@types/ember__template": "^3.16.1",
-    "@types/ember__test": "^3.16.1",
-    "@types/ember__test-helpers": "^2.0.1",
-    "@types/ember__utils": "^3.16.2",
-    "@types/faker": "^5.5.8",
-    "@types/htmlbars-inline-precompile": "^1.0.1",
-    "@types/qunit": "^2.11.2",
-    "@types/rsvp": "^4.0.4",
-    "babel-eslint": "^10.1.0",
-    "broccoli-asset-rev": "^3.0.0",
-    "ember-addon-automated-ci": "2.1.1",
-    "ember-auto-import": "2.2.0",
-    "ember-cached-decorator-polyfill": "^0.1.4",
-    "ember-cli": "~3.28.1",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "^4.0.2",
-    "ember-could-get-used-to-this": "^1.0.1",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.2",
-    "ember-maybe-import-regenerator": "^1.0.0",
-    "ember-page-title": "^6.2.2",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.2",
-    "ember-resources": "^3.2.1",
-    "ember-source": "~3.28.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.9.0",
-    "ember-try": "^1.4.0",
+    "@commitlint/cli": "^17.0.1",
+    "@commitlint/config-conventional": "^17.0.0",
+    "@nullvoxpopuli/eslint-configs": "^2.2.19",
+    "concurrency": "^0.1.4",
     "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.5",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^6.2.0",
-    "faker": "^5.5.3",
-    "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.4.1",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^2.0.0",
-    "semantic-release": "^17.4.7",
-    "typescript": "^4.4.3",
-    "webpack": "^5.58.1"
-  },
-  "release": {
-    "branches": [
-      "main",
-      "master"
-    ],
-    "plugins": [
-      "@semantic-release/commit-analyzer",
-      "@semantic-release/release-notes-generator",
-      "@semantic-release/changelog",
-      "@semantic-release/npm",
-      "@semantic-release/github",
-      "@semantic-release/git"
-    ]
+    "prettier": "^2.6.2",
+    "turbo": "^1.2.14",
+    "typescript": "4.5.2"
+  },
+  "packageManager": "pnpm@7.1.2",
+  "pnpm": {
+    "overrides": {
+      "@embroider/addon-shim": "1.7.1",
+      "@embroider/util": "1.7.1",
+      "@types/eslint": "7.29.0"
+    },
+    "peerDependencyRules": {
+      "ignoreMissing": [
+        "ember-source",
+        "ember-auto-import"
+      ]
+    }
   },
   "volta": {
-    "node": "14.17.6",
-    "yarn": "1.22.11"
-  },
-  "engines": {
-    "node": "12.* || 14.* || >= 16"
-  },
-  "ember": {
-    "edition": "octane"
-  },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "node": "16.15.0",
```

Comment:
Should `pnpm` version be here also?

Reply:
I would prefer that!
but volta doesn't yet support pnpm: https://github.com/volta-cli/volta/issues/737